Ginger/Ginger/SolutionWindows/TreeViewItems/BusinessFlowTreeItem.cs

```diff
@@ -80,6 +80,7 @@ Page ITreeViewItem.EditPage()
         {
             if (mBusinessFlowPage == null)
             {
+                mBusinessFlow.StartDirtyTracking();
```

Comment:
StartDirtyTracking() should not be added to all TreeItems one by one, it should be done in generic way.
you should do it on the ITreeViewItem.NodeObject() when Edit page is been called in generic way.